pyccl/tests/test__caching.py

```diff
@@ -0,0 +1,146 @@
+# We use double underscore to make it the first test alphabetically.
```

Comment:
I think there's one too many `_` in this file's name

Reply:
Just checked, it has 2 underscores. This is intentional, as the comment reads. It is for this test to be the first one alphabetically, so it is run first.

Reply:
Kind of embarrassing I didn't read the comment